README.md

```diff
@@ -1,3 +1,5 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/8wgCKhpZ)
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15706606&assignment_repo_type=AssignmentRepo)
 # se-day-2-git-and-github
 ## Explain the fundamental concepts of version control and why GitHub is a popular tool for managing versions of code. How does version control help in maintaining project integrity?
 
```

Comment:
Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later. 
A repository (or repo) is a storage location for software packages. It contains all the project files and the history of changes made to those files.
 A commit is a snapshot of your repository at a specific point in time. Each commit has a unique ID and includes a message describing the changes.
 Branches allow you to diverge from the main line of development and continue to work without affecting that main line. This is useful for developing features or fixing bugs.
 Merging is the process of combining changes from different branches. This helps integrate new features or fixes into the main codebase.
 A pull request is a way to propose changes to a repository. It allows others to review and discuss the changes before they are merged.